preprocessy/feature_selection/_selectKBest.py

```diff
@@ -53,6 +53,9 @@ def __init__(self):
         self.X = None
         self.y = None
 
+    def __repr__(self):
+        return f"SelectKBest(score_func={self.score_func},k={self.k}, scores={self.scores}, pvalues={self.pvalues})"
```

Comment:
I feel the default values can be directly put here, otherwise it will print the actual values of all the arguments passed by user.